test/e2e/instrumentation/events.go

```diff
@@ -172,11 +176,15 @@ var _ = common.SIGDescribe("Events API", func() {
 
 		ginkgo.By("listing events in all namespaces")
 		foundCreatedEvent = eventExistsInList(clientAllNamespaces, f.Namespace.Name, eventName)
-		framework.ExpectEqual(foundCreatedEvent, false, "should not have found test event after deletion")
+		if foundCreatedEvent {
+			framework.Failf("Should not have found test event %s in namespace %s, in list with cluster scope after deletion", eventName, f.Namespace.Name)
+		}
 
 		ginkgo.By("listing events in test namespace")
 		foundCreatedEvent = eventExistsInList(client, f.Namespace.Name, eventName)
-		framework.ExpectEqual(foundCreatedEvent, false, "should not have found test event after deletion")
+		if !foundCreatedEvent {
```

Comment:
Here the logic of the check got inverted, hence the test failures.

Reply:
Got it